concrete/config/concrete.php

```diff
@@ -603,6 +603,11 @@
             'get_available_updates' => 'http://www.concrete5.org/tools/update_core',
             'inspect_update' => 'http://www.concrete5.org/tools/inspect_update',
         ],
+        // Set to true to skip checking if there's a newer core version available (useful for example if the core is upgraded via composer) 
+        'skip_core' => false,
```

Comment:
You use this variable in one place only and that's the updater file. Shouldn't this be used for the Dashboard (Single) Page too?

Reply:
> You use this variable in one place only and that's the updater file. Shouldn't this be used for the Dashboard (Single) Page too?

@ramonleenders Yes, this new `concrete.updates.skip_core` setting is only used in `Update::getLatestAvailableUpdate()` because this is the only method that actually checks if there are new core versions. So, the dashboard page too won't report that there's a new core version.